files/jasny.bootstrap/update.json

```diff
@@ -0,0 +1,9 @@
+{
+    "packageManager": "github",
+    "name": "jasny.bootstrap",
+    "repo": "jasny/bootstrap",
+    "files": {
+        "basePath": "dist",
+        "include": ["js/jasny-bootstrap.min.js", "css/jasny-bootstrap.min.css"]
```

Comment:
You need to add dots otherwise the structure wont be retained.
`./js/jasny-bootstrap.min.js`